### DISCUSSION TITLE
Failure to use provisioning profile iOS signing

### DISCUSSION THREAD

**Discussion by bosgid** [codemagic-ci-cd/codemagic-docs#2652] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Trigger build
2. Build fails, can't find provisioning profile

### Expected results

Expect that the build will use the provisioning profile found in the previous build step.

### Actual results

Build can't find the profile even though the previous step shows that it was found.

This is the output (IDs hidden) of the first script which gets the provisioning profile (`xcode-project use-profiles`):

```

Configure code signing settings
Searching for files matching /Users/builder/Library/MobileDevice/Provisioning Profiles/*.mobileprovision
Searching for files matching /Users/builder/Library/MobileDevice/Provisioning Profiles/*.provisionprofile
List available code signing certificates in keychain /Users/builder/Library/codemagic-cli-tools/keychains/[...].keychain-db
Searching for files matching /Users/builder/clone/**/*.xcodeproj
Completed configuring code signing settings
 - Using profile "WorkWeek Internal" [...] for target "Runner" [Debug-internal] from project "Runner"
 - Using profile "WorkWeek Internal" [...] for target "Runner" [Profile-internal] from project "Runner"
 - Using profile "WorkWeek Internal" [...] for target "Runner" [Release-internal] from project "Runner"
Generated options for exporting the project
 - Method: app-store
 - Provisioning Profiles:
     - africa.workweek.employee.internal: WorkWeek Internal
 - Signing Certificate: Apple Distribution
 - Signing Style: manual
 - Team Id: ...
Saved export options to /Users/builder/export_options.plist
```
But then the build fails:

```

⠋ Fetching apps...
✓ Fetching apps (0.3s)
⠋ Building ipa with Flutter 3.22.3 (9015e1b42a)...
✗ Failed to build:
    No Accounts
    No profiles for 'africa.workweek.employee.internal' were found (131.8s)
```

### Build id (optional)

66a3630f76cec081a6d126b7

**Comment by dtrdic** [comment, marked as answer]
Note: solved over Discord.
user needed to add this parametar to shorebird release command; 
`-- --export-options-plist=/Users/builder/export_options.plist`
